src/Compilers/Core/Portable/Syntax/SyntaxTriviaList.cs

```diff
@@ -72,21 +72,27 @@ public static SyntaxTriviaList Create(ReadOnlySpan<SyntaxTrivia> trivias)
             if (trivias.Length == 0)
                 return default;
 
-            if (trivias.Length == 1)
-                return new SyntaxTriviaList(trivias[0]);
-
             return new SyntaxTriviaList(token: default, CreateNodeFromSpan(trivias), position: 0, index: 0);
         }
 
         private static GreenNode? CreateNodeFromSpan(ReadOnlySpan<SyntaxTrivia> trivias)
         {
-            if (trivias == null)
-                return null;
+            switch (trivias.Length)
+            {
+                // Also handles case where trivias is `null`.
```

Comment:
💭 `ReadOnlySpan<T>` can be empty, but not 'null'

Reply:
It can be.  And it can be tested if it is null. When null it will still have a zero length. But you get a distinctly different instance that can be tested to see if it is null vs non-null.